tests/prototype/chromium.e2e.runner.js

```diff
@@ -17,4 +18,5 @@ async function loadChromiumDriver () {
     return driver;
 };
 
-rayCharles.runTests(await loadChromiumDriver());
+// rayCharles.runTests(await loadChromiumDriver());
+stackexchange.runTests(await loadChromiumDriver());
```

Comment:
I presume the Ray Charles test is temporarily disabled here, and you'll re-enable it later?